app/grandchallenge/verifications/admin.py

```diff
@@ -17,9 +17,10 @@
     permissions=("change",),
 )
 def mark_verified(modeladmin, request, queryset):
-    queryset.filter(email_is_verified=True).update(
-        is_verified=True, verified_at=now()
-    )
+    for verfication_request in queryset:
```

Comment:
Variable names should follow their class names. So this should be `verification`, not `verification_request`.

Reply:
Noted, and fixed.